app/konnect/analytics/reference.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Reports Reference
+content_type: reference
+badge: plus
+---
+
+## Metrics
+
+Metric | Description
+-------|------------
+Request count | Total number of API calls within the selected time frame.
+Requests per minute | Number of API calls per minute within the selected time frame.
+Response latency | The amount of time, in milliseconds, that it takes to process an API request. The time starts when {{site.base_gateway}} receives a request and ends when it forwards the response back to the original caller. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response latency of 10 milliseconds means that every 1 in 100 requests took at least 10 milliseconds from request received until response returned. 
+Request size | The size of the request payload received from the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile request size of 100 bytes means that the payload size for every 1 in 100 requests was at least 100 bytes.
+Response size | The size of the response payload returned to the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response size of 100 bytes means that the payload size for every 1 in 100 response back to the original caller was at least 100 bytes.
+
+{:.note}
+> **Note:** {{site.konnect_saas}} Analytics uses percentiles to enable you to understand the real performance characteristics of your APIs. Percentiles depict a more accurate picture of what most end users experience using your API instead of hiding critical experiences by displaying averages.
+
+## Grouping and filtering
+
+When selecting a category to filter or group by, you have the following options:
+
+Category | Description
+---------|------------
+None | Aggregate all of the data in the organization without any grouping.
+Service | Group or filter the data by {{site.konnect_short_name}} service.
+Service Version | Group or filter the data by {{site.konnect_short_name}} service version. <br><br>If you select multiple service versions in a report, the report shows the sum of requests for all selected versions broken down by service. It **does not** show data points for individual service versions.
+Route | Group or filter the data by route. Route names are composed of elements referencing their related service, service version, and runtime group. See [Route entity format](#route-entity-format) for details.
+Application | Group or filter the data by application.
+Status Code | Group or filter the data by individual response status code.
+Status Code (grouped) | Group or filter the data by response status code category: 1XX, 2XX, 3XX, 4XX, and 5XX.
+
+### Route entity format
```

Comment:
We need to extend this section to not only cover how we produce the Route name but also a Service Version. Service Version are also not unique across Konnect. Hence, we had to come up with a similar approach.

For Service Version, we do:

```
KONNECT_SERVICE_NAME - VERSION (RUNTIME GROUP)
```

Examples:
![image](https://user-images.githubusercontent.com/498114/216618901-01323c17-0ab2-4b92-a786-1db3d3f2c5cb.png)